features/css-supports.yml

```
@@ -0,0 +1,7 @@
+name: CSS.supports()
+description: The `CSS.supports()` static method returns whether the browser supports a given CSS feature.
```

Maybe we can be more specific here? I don't know if this is too much:

```suggestion
description: The `CSS.supports()` static method returns whether the browser supports a CSS declaration (given by two arguments, a property and value) or an `@supports` at-rule condition string.
```